docs/jwst/cube_build/main.rst

```diff
@@ -157,7 +168,7 @@ WMAP     3      2 spatial and 1 spectral  integer
 =======  =====  ========================  =========
 
 The SCI image contains the surface brightness of cube spaxels in units of MJy/steradian. The wavelength dimension of the IFU cube
-can either be linear or non-linear. If the wavelength is non-linear then the IFU cube contain data from more than one band.  A
+can either be linear or non-linear. If the wavelength is non-linear then the IFU cube contains data from more than one band.  A
```

Comment:
Really minor nit, but shouldn't if-then clauses be punctuated in the middle with a comma, i.e. "If the wavelength is non-linear, then the IFU cube ..."?

Reply:
Not sure, but fixed.